css/_breakpoints.scss

```diff
@@ -21,7 +21,7 @@ $mq-sync:
   usn-large-medium   $usn-large-medium,
   usn-large          $usn-large,
   usn-x-large        $usn-x-large
-;
+ !default;
```

Comment:
Does this `!default~ have to be split across usn-large, usn-large-medium, et al? Or does just putting it here apply it to all?